nifi-nar-bundles/nifi-cassandra-bundle/nifi-cassandra-services-api-nar/src/main/resources/META-INF/NOTICE

```diff
@@ -1,226 +1,235 @@
+    	a temporary self-signed X.509 certificate when the JVM does not provide the
+    	equivalent functionality.  It can be obtained at:
+
+    	  * LICENSE:
+    	    * license/LICENSE.bouncycastle.txt (MIT License)
+    	  * HOMEPAGE:
+    	    * http://www.bouncycastle.org/
+
+    	This product optionally depends on 'Snappy', a compression library produced
+    	by Google Inc, which can be obtained at:
+
+    	  * LICENSE:
+    	    * license/LICENSE.snappy.txt (New BSD License)
+    	  * HOMEPAGE:
+    	    * https://github.com/google/snappy
+
+    	This product optionally depends on 'JBoss Marshalling', an alternative Java
+    	serialization API, which can be obtained at:
+
+    	  * LICENSE:
+    	    * license/LICENSE.jboss-marshalling.txt (GNU LGPL 2.1)
+    	  * HOMEPAGE:
+    	    * http://www.jboss.org/jbossmarshalling
+
+    	This product optionally depends on 'Caliper', Google's micro-
+    	benchmarking framework, which can be obtained at:
+
+    	  * LICENSE:
+    	    * license/LICENSE.caliper.txt (Apache License 2.0)
+    	  * HOMEPAGE:
+    	    * https://github.com/google/caliper
+
+    	This product optionally depends on 'Apache Commons Logging', a logging
+    	framework, which can be obtained at:
+
+    	  * LICENSE:
+    	    * license/LICENSE.commons-logging.txt (Apache License 2.0)
+    	  * HOMEPAGE:
+    	    * http://commons.apache.org/logging/
+
+    	This product optionally depends on 'Apache Log4J', a logging framework, which
+    	can be obtained at:
+
+    	  * LICENSE:
+    	    * license/LICENSE.log4j.txt (Apache License 2.0)
+    	  * HOMEPAGE:
+    	    * http://logging.apache.org/log4j/
+
+    	This product optionally depends on 'Aalto XML', an ultra-high performance
+    	non-blocking XML processor, which can be obtained at:
+
+    	  * LICENSE:
+    	    * license/LICENSE.aalto-xml.txt (Apache License 2.0)
+    	  * HOMEPAGE:
+    	    * http://wiki.fasterxml.com/AaltoHome
+
+    	This product contains a modified version of 'HPACK', a Java implementation of
+    	the HTTP/2 HPACK algorithm written by Twitter. It can be obtained at:
+
+    	  * LICENSE:
+    	    * license/LICENSE.hpack.txt (Apache License 2.0)
+    	  * HOMEPAGE:
+    	    * https://github.com/twitter/hpack
+
+    	This product contains a modified portion of 'Apache Commons Lang', a Java library
+    	provides utilities for the java.lang API, which can be obtained at:
+
+    	  * LICENSE:
+    	    * license/LICENSE.commons-lang.txt (Apache License 2.0)
+    	  * HOMEPAGE:
+        * https://commons.apache.org/proper/commons-lang/
+
+        This product contains a forked and modified version of Tomcat Native
+
+          * LICENSE:
+            * ASL2
+          * HOMEPAGE:
+            * http://tomcat.apache.org/native-doc/
+            * https://svn.apache.org/repos/asf/tomcat/native/
+
+  (ASLv2) Guava
+    The following NOTICE information applies:
+      Guava
+      Copyright 2015 The Guava Authors
+
+  (ASLv2) Dropwizard Metrics
```

Comment:
This file had CRLF endings on master, this PR has converted them to LF endings.
Dropwizard Metrics NOTICE was also added here.